airflow/www/static/js/dag/Main.tsx

```diff
@@ -31,11 +31,10 @@ import Grid from "./grid";
 import FilterBar from "./nav/FilterBar";
 import LegendRow from "./nav/LegendRow";
 import useToggleGroups from "./useToggleGroups";
-import useSelection from "./useSelection";
 
 const detailsPanelKey = "hideDetailsPanel";
 const minPanelWidth = 300;
-const collapsedWidth = "28px";
```

Comment:
This was intentional, to show just one dag run and still navigate around the task instances

Reply:
MB. Didn't notice :) 
Makes sense. 
Reverted!